### PR TITLE
code clean up for cpplint

### PR DESCRIPTION
**Pull request by edgarriba**

### CODE REVIEW

**Comment by edgarriba** [comment]
@bhack @Randl should we impose precommit ? now all the codes ar ein clangformat and pass cpplint

**Comment by bhack** [comment]
Impose a precommit doesn't assure you on the format. We can require it but probably we will need to execute also on Travis

**Comment by edgarriba** [comment]
@bhack @Randl review this

File: examples/benchmarks/main.cpp [hunk 1]
```diff
@@ -10,31 +10,26 @@
...
 
-using namespace tiny_dnn;
-using namespace std;
-
-int main(int argc, char **argv) {
-  CNN_UNREFERENCED_PARAMETER(argc);
-  CNN_UNREFERENCED_PARAMETER(argv);
-  models::alexnet nn;
+int main() {
+  tiny_dnn::models::alexnet nn;
```
Comment (karandesai-96) [inline examples/benchmarks/main.cpp]: Aren't we following `using namepsace tiny_dnn;` in examples or sample code snippets ? I have always seen it everywhere whenever there is a sample boilerplate code in docs....

Comment (edgarriba) [inline examples/benchmarks/main.cpp]: it's not recommended to use namespaces

**Comment by edgarriba** [comment]
@bhack @karandesai-96 @Randl gonna merge this in a few hours

**Comment by bhack** [comment]
It hard to review all this automatic checks at once. Probably we could merge if you have simply calm down cppcheck.

**Comment by edgarriba** [comment]
@bhack don't clearly understand your comment. You mean cppcheck as the tool itself ? Cause I've tried to run clang-tidy and I'd take a LONG time to fix everythig. Probably not using it properly

**Comment by bhack** [comment]
I meant cpplint

**Comment by jnorwood** [comment]
https://github.com/tiny-dnn/tiny-dnn/blame/master/tiny_dnn/core/kernels/tiny_quantized_matmul_kernel.h#L132

is broken due to min_a and max_a being undefined.  This is with USE_GEMMLOWP=ON
It appears to be associated with this check-in

**Comment by jnorwood** [comment]
https://github.com/tiny-dnn/tiny-dnn/blame/master/tiny_dnn/core/backend_tiny.h#L79

Commenting out the params_f_ declaration in this check-in causes backend_tiny.h:309:10 error when USE_GEMMLOWP=ON

**Pull request merged**